keyboards/dz60/rules.mk

```
@@ -55,4 +55,4 @@ BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
 
-LAYOUTS = 60_ansi 60_iso 60_hhkb
+LAYOUTS = 60_ansi 60_iso 60_hhkb 60_ansi_split_space_bs
```

This change should be reverted. This rule is only for the [Community Layouts] feature. There is no `60_ansi_split_space_bs` folder in `layouts/default/`, so this change is not needed.

```suggestion
LAYOUTS = 60_ansi 60_iso 60_hhkb
```

[Community Layouts]: https://docs.qmk.fm/#/feature_layouts

---

(at least, not yet?)

---

:point_up_2: